README.md

```diff
@@ -1,5 +1,4 @@
-Pelican Redirect: A Plugin for Pelican
-====================================================
+# Pelican Redirect: A Plugin for Pelican
```

Comment:
Apologies for the markdown re-format. It is startlingly hard to convince my editor not to re-format code on save. I can undo this if you insist.

Reply:
Nope! It's fine with me. Is this ready for another review?

Reply:
yep!